src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import Application from './containers/Application/index.js';
+import Application from './containers/Application/';
```

Comment:
Much better. Could you remove "slash" in the end?

Reply:
removed slashes where they are not needed. updated in last commit